bigbone/src/main/kotlin/social/bigbone/api/entity/streaming/ParsedStreamEvent.kt

```diff
@@ -0,0 +1,133 @@
+package social.bigbone.api.entity.streaming
+
+import kotlinx.serialization.json.Json
+import social.bigbone.JSON_SERIALIZER
+import social.bigbone.api.entity.Announcement
+import social.bigbone.api.entity.Conversation
+import social.bigbone.api.entity.Notification
+import social.bigbone.api.entity.Status
+
+/**
+ * Stream events emitted by the Mastodon API websocket stream for each event type that can occur.
+ * This is the parsed variant of the [RawStreamEvent.eventType] and [RawStreamEvent.payload]
+ * turned into easily usable data classes and objects for type-specific consumption.
+ */
+sealed interface ParsedStreamEvent {
+
+    /**
+     * A new Status has appeared.
+     * Payload contains a Status cast to a string.
+     * Available since v1.0.0
```

Comment:
More a style question. Should we clarify that we are referring to a Mastodon server version here, not our own versioning?

Reply:
That would probably make sense, yeah. I’ll add it.